modules/imgcodecs/src/grfmt_gdal.cpp

```diff
@@ -541,6 +541,13 @@ void GdalDecoder::close(){
     m_driver = NULL;
 }
 
+/**
+ * try to go to next page
+*/
+bool GdalDecoder::nextPage(){
```

Comment:
i think GDAL driver loads only first page of multipage images. i will try to implement `GdalDecoder::nextPage()` after actual changes seems good.